tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -157,63 +143,51 @@ def _create_directory_stack(
-        "ad_users_base_search": ad_users_base_search,
-        "ad_admin_password": ad_admin_password,
-        "ad_user_password": ad_user_password,
-        "ad_domain_name": ad_domain_name,
-        "ad_domain_short_name": ad_domain_short_name,
-        "default_ec2_domain": "ec2.internal" if region == "us-east-1" else f"{region}.compute.internal",
-        "ad_admin_user": "Administrator" if directory_type == "SimpleAD" else "Admin",
-        "directory_type": directory_type,
-    }
     logging.info("Creating stack %s", directory_stack_name)
-    with open(render_jinja_template(directory_stack_template_path, **config_args)) as directory_stack_template:
+
+    with open(directory_stack_template_path) as directory_stack_template:
         private_subnet = vpc_stack.get_private_subnet()
         private_subnets = vpc_stack.get_all_private_subnets().copy()
         private_subnets.remove(private_subnet)
 
-        params = [
+        users = ""
+        for i in range(NUM_USERS_TO_CREATE):
+            users += f"PclusterUser{i},"
+
+        stack_parameters = [
+            {
+                "ParameterKey": "DomainName",
+                "ParameterValue": f"{directory_type.lower()}.{random_alphanumeric(size=10)}.multiuser.pcluster",
+            },
+            {"ParameterKey": "AdminPassword", "ParameterValue": ADMIN_PASSWORD},
+            {
+                "ParameterKey": "ReadOnlyPassword",
+                "ParameterValue": "".join(random.choices(string.ascii_letters + string.digits, k=60)),
+            },
+            {"ParameterKey": "UserNames", "ParameterValue": users[:-1]},
+            {"ParameterKey": "UserPassword", "ParameterValue": USER_PASSWORD},
+            {"ParameterKey": "DirectoryType", "ParameterValue": directory_type},
             {"ParameterKey": "Vpc", "ParameterValue": vpc_stack.cfn_outputs["VpcId"]},
             {"ParameterKey": "PrivateSubnetOne", "ParameterValue": private_subnet},
+            {"ParameterKey": "PrivateSubnetTwo", "ParameterValue": private_subnets[0]},
+            {"ParameterKey": "Keypair", "ParameterValue": request.config.getoption("key_name")},
             {
-                "ParameterKey": "PrivateSubnetTwo",
-                "ParameterValue": private_subnets[0],
+                "ParameterKey": "AdminNodeAmiId",
+                "ParameterValue": "/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2",
             },
-            {"ParameterKey": "PublicSubnetOne", "ParameterValue": vpc_stack.get_public_subnet()},
         ]
         directory_stack = CfnStack(
             name=directory_stack_name,
             region=region,
             template=directory_stack_template.read(),
-            parameters=params,
-            capabilities=["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"],
+            parameters=stack_parameters,
+            capabilities=["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM", "CAPABILITY_AUTO_EXPAND"],
```

Comment:
Because the template uses the AWS::Serverless transform macro which requires the capability. From https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_UpdateStack.html:

"If your stack template contains one or more macros, and you choose to update a stack directly from the processed template, without first reviewing the resulting changes in a change set, you must acknowledge this capability."